media/linux/pds-sqlite3-queries/create-training-rosters.py

```diff
@@ -54,11 +54,24 @@ def pretty_member(member):
 
     email = PDSChurch.find_any_email(member)[0]
 
+    mail1 = member['family']['StreetAddress1']
+    mail2 = member['family']['StreetAddress2']
+    mail4 = member['family']['StreetZip']
+    mail = f'{mail1} {mail2}, {mail4}'
+
+    if (int)(member['family']['ParKey']) > 9000:
+        active = "No"
+    else:
+        active = "Yes"
+
     m = {
         'mid'       :   member['MemRecNum'],
         'name'      :   member['first']+' '+member['last'],
         'phones'    :   phones,
-        'email'     :   email,}
+        'email'     :   email,
+        'active'    :   active,
+        'mail'      :   mail
```

Comment:
FWIW, it's probably good practice -- in Python, at least -- to put a comma after ever element in a dictionary initialization like this (just in case you add more initializing members someday -- Future Harrison will thank you).